EvidenceApi.Tests/V1/E2ETests/EvidenceRequestsTest.cs

```diff
@@ -40,10 +40,10 @@ public async Task CanCreateEvidenceRequestWithValidParams()
             }";
 
             var jsonString = new StringContent(body, Encoding.UTF8, "application/json");
-            var response = await Client.PostAsync(uri, jsonString).ConfigureAwait(true);
```

Comment:
I've removed the `.ConfigureAwait(true)` from the relevant files because we're suppressing the build error, so this is not needed.